packages/plugins/src/plugins/acrossv2/constants.ts

```diff
@@ -0,0 +1,24 @@
+import { NetworkId } from '@sonarwatch/portfolio-core';
+import { Address } from 'viem';
+
+// Yes the id is wrong, it's acrossv2 but debank one is acrocc_v2
+export const platformId = 'acrocc_v2';
+export const platform = {
+  id: platformId,
+  name: 'acrocc_v2',
```

Comment:
Shouldn't we re-use the constant to avoid copy-pasta ?

```suggestion
  name: platformId,
```